tests/src/utils.rs

```diff
@@ -511,6 +511,16 @@ pub async fn run_ls(args: Option<Vec<&str>>, namespace: Option<&str>) -> TestPro
     run_mirrord(mirrord_args, Default::default()).await
 }
 
+/// Runs `mirrord verify-config [--ide] "/path/config.json"`.
```

Comment:
Add a note to not modify this test as this implies breaking change, if there's new feature add a new test.